package.json

```diff
@@ -181,7 +182,7 @@
     "json-stringify-safe": "5.0.1",
     "lodash": "^4.17.21",
     "lru-cache": "^4.1.5",
-    "minimatch": "^3.0.4",
+    "minimatch": "3.0.5",
```

Comment:
maybe `^3.0.5` is better even though 3.0.5 might the latest.
^version will update to all future minor/patch versions, without incrementing the major version